docs/contributing/managing-pull-requests.md

```diff
@@ -0,0 +1,178 @@
+
+- Correctness — whether the code does what we think it does
+- Tests — when fixing a bug or adding a new feature, it can be very valuable to add tests. While we do merge some small PRs without them, more often than not, it's good to have tests asserting behaviour! This can be a combination of unit tests for the specific package, snapshot tests and end to end tests. The goal here is to ensure that something that is being fixed or added _remains_ fixed or working the way we expect it to. Good tests ensure this!
+- Code Quality — while it isn't good to nit pick, reasonable changes that improve readability are great to point out
+- Documentation in the package's README if you're adding something
+
+#### Starters or Site Showcase
+
+For PRs that add a site or a starter to the showcase, we ought to check:
+
+- We check if the site or starter is indeed a Gatsby site or starter respectively
+- Links — check if the links are working and accessible
+- Tags — ensure the tags are not arbitrary
+- Featured Status — for the site showcase, we like to default to _not_ featuring a site and typically someone from the Gatsby team features it later if they like!
+
+#### Blog posts
+
+For PRs that add a blog posts, we ought to check:
+
+- Correctness — whether the added documentation is technically correct
+- Style — whether the written language follows our [style guide](https://www.gatsbyjs.org/contributing/gatsby-style-guide/)
+- Blog posts are similar to documentation in that they're long form writing, but also want to ensure that blog posts on [gatsbyjs.org](https://www.gatsbyjs.org) aren't purely promotional, spammy or inappropriate.
+
+## Automated Checks
+
+Our repository on [GitHub](https://github.com/gatsbyjs/gatsby) has several automated CI checks that are run automatically for all PRs. These include tests, linting and even preview builds [gatsbyjs.org](https://www.gatsbyjs.org).
+
+We want all of these checks to pass. While it's okay to review a work in progress PR with some failed checks, a PR is only ready to ship when everything is green.
+
+Let's go over some common failure cases in our cases and how to approach fixing them:
+
+### Linting
+
+We lint all code and documentation pretty aggressively (consistency is good) and while that is great, you might find some PRs failing the linting check.
+
+An easy fix for this is to run:
+
+> 💡 npm run format
+
+This will format all code and documentation and should get rid of those pesky linting issues!
+
+## Other Checks
+
+### Testing Locally
+
+While we have many _many_ tests in our repository (that are run automatically on every commit), there can be times when there exists an edge case (or five) that isn't covered by these.
+
+In situations like this, testing the change locally can be very valuable.
+
+> 💡 In case this is the first time you're doing this, you might have to [set up your development environment](https://www.gatsbyjs.org/contributing/setting-up-your-local-dev-environment).
```

Comment:
```suggestion
> 💡 In case this is the first time you're doing this, you might have to [set up your development environment](/contributing/setting-up-your-local-dev-environment).
```